mayavi/tools/data_wizards/csv_sniff.py

```diff
@@ -171,7 +171,9 @@ def loadtxt(self):
         """ Return the array (by using numpy.loadtxt), using the sniffed
             information in the keyword arguments.
         """
-        return loadtxt(self._filename, **self.kwds())
+        with open(self._filename) as fh:
```

Comment:
To keep the same behaviour as before, I think we want to specify `encoding="latin1"` here: see https://github.com/numpy/numpy/blob/v1.21.0/numpy/lib/npyio.py#L1066-L1070.